android/BOINC/app/src/main/java/edu/berkeley/boinc/PrefsFragment.java

```diff
@@ -407,7 +410,8 @@ public void onClick(View v) {
          	   } else if(item.ID == R.string.prefs_network_daily_xfer_limit_mb_header ||
          			   item.ID == R.string.battery_temperature_max_header ||
          			   item.ID == R.string.prefs_disk_min_free_gb_header ||
-         			   item.ID == R.string.prefs_disk_access_interval_header) {
+         			   item.ID == R.string.prefs_disk_access_interval_header ||
+					   item.ID == R.string.prefs_other_store_at_least_x_days_of_work_header) {
```

Comment:
Please remove the additional indentation (one level).

Reply:
Done.

Reply:
Done

Reply:
Now should be fine.